.gitignore

```diff
@@ -1,4 +1,5 @@
 .buildpath
+.phpunit.result.cache
```

Comment:
Since the `PHPUnit 8` is executed, the `.phpunit.result.cache` will be generated.

And this makes sens to add this file to the `.gitignore` file.